runtime/near-vm-logic/Cargo.toml

```diff
@@ -24,19 +24,20 @@ sha3 = ">=0.8,<0.10"
 near-runtime-fees = { path = "../near-runtime-fees", version = "2.3.0" }
 near-vm-errors = { path = "../near-vm-errors", version = "2.3.0" }
 near-runtime-utils = { path = "../near-runtime-utils", version = "2.3.0" }
+near-primitives = { path = "../../core/primitives", version = "0.1.0"}
```

Comment:
`near-vm-logic` and `near-runtime-fees` are dependencies of near-sdk-rs which we MUST maintain compilable for Wasm target (and slim) for smart contracts, so they cannot depend on near-primitives.

This is why you observe some code duplication.

Reply:
I thought this is the point. We want to publish `near-primitives` crate

Reply:
@evgenykuzyakov

1. I don't believe it will compile near-crypto, sha2, rand, jemalloc for Wasm target
2. We don't help guys porting near-sdk-rs for `no_std` environment which such a change...